### PR TITLE
Add support for environment variables

### LINKED ISSUE #1203: Fix the css style is in dashboard..
Fix the css style is in dashboard..

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Fixes #1203

### CODE REVIEW

**Comment by patrys** [comment]
Do we need two separate variables for this?

**Comment by dominik-zeglen** [comment]
@patrys actually... no.

File: common.env [hunk 1]
```diff
@@ -5,3 +5,5 @@ OPENEXCHANGERATES_API_KEY
 REDIS_URL=redis://redis:6379/0
 REDIS_BROKER_URL=redis://redis:6379/1
 SECRET_KEY=changeme
+
+ASSETS_PRODUCTION=/static/assets/
```
Comment (patrys) [inline common.env]: Please rename this `STATIC_URL` to match Django's nomenclature.

File: webpack.config.js [hunk 2]
```diff
@@ -4,18 +4,21 @@ var FaviconsWebpackPlugin = require('favicons-webpack-plugin');
...
 var autoprefixer = require('autoprefixer');
+var dotenv = require('dotenv');
 
 var resolve = path.resolve.bind(path, __dirname);
 
 var extractTextPlugin;
 var fileLoaderPath;
 var output;
 
+dotenv.config({path: 'common.env'});
```
Comment (patrys) [inline webpack.config.js]: Do not parse any config files. Environment variables are _the_ API.

File: docs/configuration.rst [hunk 3]
```diff
@@ -51,4 +51,7 @@ Environment variables
 
 
 ``MAX_CART_LINE_QUANTITY``
-  Controls maximum number of items in one cart line. Defaults to ``50``
+  Controls maximum number of items in one cart line. Defaults to ``50``.
+
+``ASSETS_PRODUCTION``
```
Comment (elwoodxblues) [inline docs/configuration.rst]: Isn't it called `STATIC_URL` now?

File: package.json [hunk 4]
```diff
@@ -47,6 +47,7 @@
     "babel-preset-react": "^6.22.0",
     "babel-preset-stage-0": "^6.22.0",
     "css-loader": "^0.28.7",
+    "dotenv": "^4.0.0",
```
Comment (elwoodxblues) [inline package.json]: I guess we don't need this anymore.

**Pull request merged**